src/main/java/cat/udl/eps/softarch/demo/domain/Batch.java

```diff
@@ -16,10 +20,19 @@ public class Batch {
     @GeneratedValue(strategy = GenerationType.AUTO)
 
     private Long id;
-    private int amount = -1;
-    private Long latitude;
-    private Long longitude;
-    private ZonedDateTime date;
+    @NotNull
+    @Min(value = 1, message = "There must be one or more batches")
```

Comment:
The amount refers to the number of individual seeds, so better to say "There must be one or more seeds"